thirdeye-ui/src/app/components/home-page/trending-anomalies/trending-anomalies.component.tsx

```diff
@@ -195,11 +196,11 @@ export const TrendingAnomalies: FunctionComponent<TrendingAnomaliesProps> = ({
                             <Typography variant="h6">
                                 {capitalize(
                                     t(
-                                        "message.there-have-been-no-recent-entity-in-the-timePeriod",
+                                        "message.there-have-been-no-entity-in-the-timePeriod",
                                         {
                                             entity: t("label.anomalies"),
                                             timePeriod: t(
-                                                "label.last-6-months"
+                                                "label.selected-time-range"
```

Comment:
Good candidate to refactor to use `EmptyStateSwitch`

Reply:
Implemented